internal/command/command.go

```diff
@@ -326,12 +326,27 @@ func createTmpWorkingRoot(t time.Time) (string, error) {
 	return path, nil
 }
 
-func commit() error {
-	return fmt.Errorf("commit is not implemented")
+// No commit is made if there are no file modifications.
+func commitAll(ctx context.Context, repo *gitrepo.Repo) error {
+	status, err := gitrepo.AddAll(ctx, repo)
+	if err != nil {
+		return err
+	}
+	if status.IsClean() {
+		slog.Info("No modifications to commit.")
+		return nil
+	}
+
+	gitrepo.PrintStatus(ctx, repo)
+	// TODO: Get commit message from googleapis
```

Comment:
For now, perhaps we could pass a message in from the caller? ("Configured API {x}", "Regenerated all APIs" or "Regenerated API {x}" would all be at least marginally better than this.